### DISCUSSION TITLE
Request to join the Maintainers teams

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [EddieHubCommunity/LinkFree#1103] [General]
I would love to be a maintainer of this project, and have requested to join the team in the organization. Please have a look and accept my request @eddiejaoude @EddieHubCommunity/linkfree

**Comment by Cahllagerfeld** [comment]
I think posting messages every 2 hours, wont increase your chances to get approved 😅

**Reply by KendallDoesCoding** [reply]
Yeah ik. I just posted it here incase yall dont get a email notification or something lol, and coz its related to this project too.

**Comment by abdussalam02** [comment]
My json file has been updated but no changes are shown in the LinkFree website

**Reply by EmmaDawsonDev** [reply]
changes to profiles are not automatically deployed but don't worry - your change will appear within the next couple of days.

**Comment by eddiejaoude** [comment]
> I would love to be a maintainer of this project, and have requested to join the team in the organization. Please have a look and accept my request

Hi Kendall, we have received the many notifications from you about becoming a maintainer, from "joining the team" GitHub request as well as the "team message" and the discussion here. Spamming the maintainers every few hours is not how people become maintainers, **please remember people are in different timezones across the world and they have other commitments like jobs, family, down time etc**. We promote open source and collaboration but we also promote good mental health, so breaks from code are important.

While we appreciate your enthusiasm, you have not been envolved in EddieHub for that long to become a maintainer, plus I think you need to make your community support/feedback/help more inclusive as not to scare people off.

Hopefully the constructive feedback will help you to continue to grow and improve as an open source contributor and maintainer.

**Reply by KendallDoesCoding** [reply]
Hey Eddie, Yes, that's cause I requested in multiple teams, projects to become a maintainer, I didn't have a intention to spam yall and put the discussion here incase your email notifics were off.

Yes, that's true. I haven't been involved in EddieHub, until recently, so I completely understand your decision

Thanks for the feedback,
KendallDoesCoding

**Reply by KendallDoesCoding** [reply]
@eddiejaoude But, wdym spamming the maintainers, I just sent requests to join teams right? I didn't ping anyone, right?

**Reply by naomi-lgbt** [reply]
Your request to join the team triggered an email for folks. Your message in the team view triggered a notification for folks. Your new discussion triggered a notification for folks.

But, IMHO what's more concerning is that you have a habit of pinging maintainers in things like new issues, comments, etc. 

<img width="429" alt="image" src="https://user-images.githubusercontent.com/63889819/153546574-1903d321-25a4-4325-9921-75a4051b3e67.png">

<img width="736" alt="image" src="https://user-images.githubusercontent.com/63889819/153546602-c476c4d8-298b-429a-ac9a-ee74053eeed5.png">

<img width="615" alt="image" src="https://user-images.githubusercontent.com/63889819/153546676-2722a12f-9dac-4b13-9dd3-ae1e1759737c.png">

**Reply by eddiejaoude** [reply]
Probably best I lock this discussion, so not more notificaitions are created on the subject.
